features/smoke_tests/handled.feature

```diff
@@ -7,9 +7,12 @@ Scenario: Notify caught Java exception with default configuration
 
     # Exception details
     And the error payload field "events" is an array with 1 elements
+    And the error payload field "events.0.device.runtimeVersions.androidApiLevel" is not null
+    And the error payload field "events.0.device.runtimeVersions.osBuild" is not null
     And the exception "errorClass" equals "java.lang.IllegalStateException"
     And the exception "message" equals "HandledJavaSmokeScenario"
     And the exception "type" equals "android"
+    And the event "context" equals "FooContext"
```

Comment:
These new assertions for context/runtimeVersions could be moved into its own sections with a commented title like breadcrumbs/metadata have been - which might make things a bit easier to read down the line.

Reply:
Done